packages/core/src/api/app/AppBuilder.tsx

```diff
@@ -87,12 +90,24 @@ export default class AppBuilder {
             );
             break;
           }
+          case 'feature-flag': {
+            registeredFeatureFlags.push({
+              pluginId: plugin.getId(),
+              name: output.name,
+            });
+            break;
+          }
           default:
             break;
         }
       }
     }
 
+    const FeatureFlags = this.apis && this.apis.get(featureFlagsApiRef);
+    if (FeatureFlags) {
+      FeatureFlags!.registeredFeatureFlags = registeredFeatureFlags;
```

Comment:
don't need the exclamation mark in here

Reply:
Fixed in 6de870fc4d9fdba8f581e4a72f000b2315bf0f7d